docs/sphinx_extensions.rst

```diff
@@ -20,3 +24,22 @@ add the following to your ``conf.py``::
 Then, in an ``.rst`` file::
 
     .. featuretoggles::
+
+.. _sphinx_settings:
+
+``settings``
+============
+
+Similar to feature toggles, Django setting annotations can also be parsed to generate human-readable documentation. Add the following to your ``conf.py``::
```

Comment:
Need to clarify that this is for Django settings that are not being used as toggles, or non-toggle Django settings.

Reply:
:ok_hand: